### DISCUSSION TITLE
/v2/zh/cookbook/tutorial/create.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#201] [Announcements]
# /v2/zh/cookbook/tutorial/create.html

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/cookbook/tutorial/create.html

**Comment by axisact** [comment]
test

**Comment by keyL-liucong** [comment]
test

**Reply by keyL-liucong** [reply]
111

**Comment by xietianhua0205** [comment]
问一下大佬，这个系统的文章更新只能在代码里面编写然后发布到服务器，没有管理模块对吧？
